src/core/lib/promise/activity.h

```diff
@@ -208,12 +222,20 @@ class Activity : private Wakeable {
   bool RefIfNonzero();
   // Drop the (proved existing) wait handle.
   void DropHandle() ABSL_EXCLUSIVE_LOCKS_REQUIRED(mu_);
+  // Set the action that occured during this run.
+  // We use max to combine actions so that cancellation overrides wakeups.
```

Comment:
I wonder if it's worth commenting about this on the enum class itself, that any newly-added value will override a smaller value when considering the overall result of multiple actions during a run. That's non-intuitive.

Reply:
Done